ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToClient.hs

```diff
@@ -205,13 +213,31 @@ defaultCodecs ccfg version networkVersion = Codecs {
     dec :: SerialiseNodeToClient blk a => forall s. Decoder s a
     dec = decodeNodeToClient ccfg version
 
+    encInter :: WithInterDefault blk -> Encoding
```

Comment:
Does this need a proper `SerialiseNodeToClient` instance and introduce a new `BlockNodeToClientVersion`?